web/src/HUD.tsx

```diff
@@ -38,24 +39,18 @@ type HudState = {
     RunningTiltBuild: TiltBuild
     LatestTiltBuild: TiltBuild
     FeatureFlags: { [featureFlag: string]: boolean }
+    RegisterTokenURL: string
   } | null
   IsSidebarClosed: boolean
   SnapshotLink: string
+  ShowSnapshotModal: boolean
```

Comment:
fine for this PR, but it seems weird that all these fields are capitalized

Reply:
They're capitalized because they come from Go and we don't override the JSON serialization.

Reply:
They're capitalized because they come from Go and we don't override the JSON serialization.

Reply:
this property doesn't come from Go, though, right? or am i misunderstanding?

Reply:
Ohh you're right @nicks, good call. I can lowercase it.